lib/authCodeGrant.js

```diff
@@ -252,19 +252,19 @@ function redirect (done) {
   var url,
     state;
 
+  if(typeof this.req.body.state !== 'undefined') {
```

Comment:
this is actually a code snippet from the original library. i have updated it nonetheless.